jck/agent-drive.sh

```diff
@@ -1,14 +1,80 @@
 #!/bin/bash
 
-echo "Starting JCK agent.."
-eval $1
+if [ $# -eq 2 ]; then
```

Comment:
could split this into two functions?

Reply:
Cleaned up code-duplication in agent-drive.sh. Organized code in functions (test: Grinder/32994).